### DISCUSSION TITLE
RoomCard and SideCard bug?

### DISCUSSION THREAD

**Discussion by juan0x00** [shannonhochkins/ha-component-kit#42] [Show and tell]
Hi there,

Thanks for implementing the EntitiesCard. It's amazing! I've been trying to create my own dashboard, but I'm having a problem closing a RoomCard. It doesn't close, and when I try to switch to another RoomCard using the sidebar, it doesn't switch either.

![ss](https://github.com/shannonhochkins/ha-component-kit/assets/63176733/58b1583e-ecf9-49cf-96a2-b441a42c707f)

At first, I thought I had made a coding mistake, but it seems that your code example doesn't work either: https://shannonhochkins.github.io/ha-component-kit/?path=/story/components-cards-sidebarcard--sidebar-example

I would also like to see an the entity list (similar to the Home Assistant entities list) as a new feature!

**Comment by shannonhochkins** [comment]
Hi! I know what the issue is, I made a change to the fabCard and the onClick isn't firing, so sorry for any inconvenience I will have a new version released today

**Comment by shannonhochkins** [comment]
I've just published a new version of components - are you able to install the latest and report back please?

**Reply by juan0x00** [reply]
I just tested it and it works!! Thank you!!! 🧐

**Reply by shannonhochkins** [reply]
Great!
